test/LRPTests/APITests.cpp

```diff
@@ -0,0 +1,164 @@
+        }
+
+        TEST_METHOD(LaunchLRP_FromStartupTask)
+        {
+            VerifyLRP_IsRunning(false);
+
+            STARTUPINFO startupInfo{};
+            ZeroMemory(&startupInfo, sizeof(startupInfo));
+            startupInfo.cb = sizeof(startupInfo);
+
+            // Build the Solution OutDir path where the Startup Task exe is.
+            // Path: $OutDir/WindowsAppRuntime.Test.Singleton.Msix/msix/PushNotificationsLongRunningTask.StartupTask.exe
+            auto startupTaskExePath{ Test::FileSystem::GetSolutionOutDirPath() / Test::Packages::WindowsAppRuntimeSingleton::c_PackageDirName };
+            startupTaskExePath += LR"(.Msix/msix/PushNotificationsLongRunningTask.StartupTask.exe)";
+
+            wil::unique_process_information processInfo;
+            BOOL wasProcessCreated{ CreateProcess(
+                startupTaskExePath.c_str(),
+                nullptr, // command line options
+                nullptr, // process attributes
+                nullptr, // thread attributes
+                FALSE,   // inherit handles
+                NORMAL_PRIORITY_CLASS, // creation flags
+                nullptr, // lpEnvironment
+                nullptr, // current directory for the process
+                &startupInfo,
+                &processInfo) };
+
+            if (wasProcessCreated == FALSE)
+            {
+                VERIFY_SUCCEEDED(GetLastError());
+            }
+
+            // Wait for the process to come up and be captured in the snapshot from verification step.
+            Sleep(1000);
+            VerifyLRP_IsRunning(true);
+        }
+
+        void VerifyLRP_IsRunning(bool isRunning)
+        {
+            wil::unique_handle processesSnapshot(CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0));
+            VERIFY_IS_NOT_NULL(processesSnapshot.get());
+
+            PROCESSENTRY32 processEntry{};
+            processEntry.dwSize = sizeof(processEntry);
+
+            BOOL result{ Process32First(processesSnapshot.get(), &processEntry) };
+            while (result != FALSE)
+            {
+                if (wcscmp(L"PushNotificationsLongRunningTask.exe", processEntry.szExeFile) == 0)
+                {
+                    VERIFY_IS_TRUE(isRunning);
+                    DWORD processId{ processEntry.th32ProcessID };
+
+                    wil::unique_handle longRunningProcessHandle(OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, processId));
+                    DWORD exitCode{};
+                    BOOL exitCodeProcess{ GetExitCodeProcess(longRunningProcessHandle.get(), &exitCode) };
+
+                    VERIFY_SUCCEEDED(exitCodeProcess == FALSE ? GetLastError() : S_OK);
+                    VERIFY_ARE_EQUAL(exitCode, STILL_ACTIVE);
+
+                    return;
+                }
+
+                result = Process32Next(processesSnapshot.get(), &processEntry);
+            }
+
+            VERIFY_IS_FALSE(isRunning);
+        }
+
+        TEST_METHOD(LaunchLRP_FromCoCreateInstance)
```

Comment:
do you need this test..I guess other tests are going to be doing this.

Reply:
Tbh I was thinking the same thing. Will remove.